ixwebsocket/IXWebSocket.cpp

```diff
@@ -73,6 +73,14 @@ namespace ix
         _perMessageDeflateOptions = perMessageDeflateOptions;
     }
 
+#ifdef IXWEBSOCKET_USE_TLS
```

Comment:
Let's get rid of the whole macro / ifdef stuff.

Reply:
yup